src/server/plugins/engine/pageControllers/SummaryPageController.ts

```diff
@@ -302,8 +298,12 @@ async function sendEmail(
 ) {
   request.logger.info(['submit', 'email'], 'Preparing email')
 
+  const { path } = request
+
+  const isPreview = path.toLowerCase().startsWith(PREVIEW_PATH_PREFIX)
```

Comment:
We're duplicating this logic. Better to pull this out into a shared function so they don't drift.

Reply:
ahh ok, i didn't see its used in `plugin.ts`. I will refactor that

Reply:
done